docs/advanced-topics/networkobject-parenting.md

```diff
@@ -60,27 +45,52 @@ If you run into a situation where your client must trigger parenting a `NetworkO
+:::tip
+You can use `NetworkObject.RemoveParent` to remove a parent from a child which doesn't require you to cast `null` to a `GameObject` or `NetworkObject` when using `NetworkObject.TrySetParent`.  This helper method was added for convenience and internally it does the casting for you:
+ ```csharp
+ public bool TryRemoveParent(bool worldPositionStays = true)
+ {
+     return TrySetParent((NetworkObject)null, worldPositionStays);
+ }
+```
 :::
 
 ### Only Spawned NetworkObjects Can Be Parented
 A `NetworkObject` can only be parented if it is spawned and can only be parented under another spawned `NetworkObject`. This also means that `NetworkObject` parenting can only occur during a network session (netcode enabled game session).  Think of `NetworkObject` parenting as a netcode event.  In order for it to happen, you must have, at very minimum, a server or host instance started and listening.
 
 ### Invalid `NetworkObject` Parenting Will Be Reverted
-If an invalid/unsupported `NetworkObject` parenting happens, Netcode will immediately revert it back to its original parenting status. 
+If an invalid/unsupported `NetworkObject` parenting happens, the attempted parenting action will be immediately reverted it back to its original parenting status. 
 
 **For example:** 
 If you had a `NetworkObject` who's current parent was root and tried to parent it in an invalid way (i.e. under a GameObject without a `NetworkObject` component) then a warning message would be logged and the `NetworkObject` would revert back to having root as its parent.
 
 ### In-scene NetworkObject parenting of players
-In-scene placed `NetworkObject` parenting of players requires the client to be synchronized first.  Since a server can only perform parenting related actions, the server must have already received the `NetworkSceneManager` generated `SceneEventType.SynchronizeComplete` message before the server can parent the client's player `NetworkObject`.
+In-scene placed `NetworkObject` parenting of players requires the client to be synchronized first.  Since a server can only perform parenting related actions, the server must have already received the `NetworkSceneManager` generated `SceneEventType.SynchronizeComplete` message  before the server can parent the client's player `NetworkObject`.
 :::info
 For more information, see the "[Real World In-scene NetworkObject Parenting of Players Solution](inscene_parenting_player.md)".
 :::
 
+### WorldPositionStays usage
+When using the `NetworkObject.TrySetParent` or `NetworkObject.TryRemoveParent` methods, the `WorldPositionStays` parameter is synchronized with currently connected and late joining clients. When removing a child from its parent, you should use the same `WorldPositionStays` value that was used to parent the child.  More specifically when `WorldPositionStays` is set to false this applies, but if you are using the defalt value of `true` then this is not required (because it is the default).
```

Comment:
> When using the `NetworkObject.TrySetParent` or `NetworkObject.TryRemoveParent` methods, the `WorldPositionStays` parameter is synchronized with currently connected and late-joining clients. You should use the same `WorldPositionStays` value that was used to parent the child when removing a child from its parent. More specifically, when `WorldPositionStays` is set to `false`, this applies, but if you are using the default value of `true`, then this is not required (because it is the default).

Reply:
Yeah... that adjustment in the middle has a better flow to it.